src/rdkafka_conf.c

```diff
@@ -739,6 +739,21 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           _RK(ssl.keystore_location),
           "Path to client's keystore (PKCS#12) used for authentication."
         },
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+        { _RK_GLOBAL, "ssl.engine.location", _RK_C_STR,
+          _RK(ssl.engine_location),
+          "Path to openssl engine library. OpenSSL >= 1.1.0 required."
```

Comment:
OpenSSL, not openssl or Openssl.

Reply:
Why is OpenSSL 1.1 required?

Reply:
ENGINE_load_ssl_client_cert from engine is the main method which loads the ssl cert. To call this method we also need to provide stack of ca cert names. To prepare that stack I can not get methods before openssl 1.1 
These are the methods from openssl : 1.1 X509_OBJECT_get0_X509 and X509_STORE_get0_objects

Reply:
Okay :+1: